doc/rados/troubleshooting/troubleshooting-osd.rst

```diff
@@ -232,10 +232,21 @@ Or::
 The best way to deal with a full cluster is to add new ``ceph-osds``, allowing
 the cluster to redistribute data to the newly available storage.
 
-If you cannot start an OSD because it is full, you may delete some data by deleting
-some placement group directories in the full OSD.
+If you cannot add a new OSD, you can do the following:
 
-.. important:: If you choose to delete a placement group directory on a full OSD,
+1. Establish rules to lock out all clients (e.g. firewall rules)
+
+2. Reweigh OSDs, if necessary, with ``ceph osd crush reweight <osd.ID> <weight>``
```

Comment:
what is the suggested weight? maybe we could recommend `ceph osd reweight-by-utilization`?